viamonvif/service.go

```diff
@@ -287,6 +312,53 @@ func downloadPreviewImage(ctx context.Context, logger logging.Logger, snapshotUR
 	return dataURL, nil
 }
 
+// fetchImageFromRTSPURL fetches the image from the rtsp URL and returns it as a data URL.
+func fetchImageFromRTSPURL(ctx context.Context, logger logging.Logger, rtspURL string) (string, error) {
+	// Wrap viamrtsp.Config in a resource.Config
+	rtspConfig := viamrtsp.Config{
+		Address: rtspURL,
+	}
+	resourceConfig := resource.Config{
+		Name:                "camera_name", // Replace with the actual camera name
```

Comment:
do we need to actually replace this with the camera name or does it not matter?

Reply:
I am no longer concerned about the actual name used here matching up with discovery.

Also, since we are not using a robot harness, I am not worried about name collisions.

Reply:
Using stock name of `tmp-camera`